### DISCUSSION TITLE
Test new discussion post

### DISCUSSION THREAD

**Discussion by GitMurf** [roamhacker/SmartBlocks#124] [Show and tell]
Test post...

How is this any different than a "Issue"?

**Comment by GitMurf** [comment]
Testing comment

**Reply by sc0ttwad3** [reply]
A **_discussion_** can be open ended as part of "the collaborative communication forum for the community around an open source project, [where] community members can ask and answer questions, share updates, have open-ended conversations, and follow along on decisions affecting the community's way of working." 

Whereas, an **_issue_** is usually associated with specific problems, "code" or content in a repository, as a pseudo trouble-ticket, and so on.

[GitHub Discussions](https://docs.github.com/en/free-pro-team@latest/discussions)
